package.json

```diff
@@ -68,6 +75,7 @@
     "phantomjs": "^1.9.17",
     "react-hot-loader": "^1.2.8",
     "react-router": "^1.0.0-rc1",
+    "rimraf": "^2.4.3",
```

Comment:
It's not, it's a devDependencies. You may have no expanded your diff.
True, but I needed Windows compatibility to test and continue developing this branch.